lib/internal/errors.js

```diff
@@ -112,6 +112,8 @@ module.exports = exports = {
 // Note: Please try to keep these in alphabetical order
 E('ERR_ARG_NOT_ITERABLE', '%s must be iterable');
 E('ERR_ASSERTION', (msg) => msg);
+E('ERR_CONSOLE_WRITABLE_STREAM',
+  (name) => `Console expects a writable stream instance for ${name}`);
```

Comment:
No special reason. I made this PR when there was no other error messages using `%s` syntax. 
Will change if you think that way is better